public/v4/apps/excalidraw.yml

```diff
@@ -0,0 +1,77 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: kiliandeca/excalidraw:$$cap_excalidraw_version
+        restart: always
+        environment:
+            BACKEND_V2_GET_URL: https://$$cap_appname-storage.$$cap_root_domain/api/v2/scenes/
+            BACKEND_V2_POST_URL: https://$$cap_appname-storage.$$cap_root_domain/api/v2/scenes/
+            LIBRARY_URL: https://libraries.excalidraw.com
+            LIBRARY_BACKEND: https://us-central1-excalidraw-room-persistence.cloudfunctions.net/libraries
+            SOCKET_SERVER_URL: https://$$cap_appname-room.$$cap_root_domain/
+            STORAGE_BACKEND: 'https'
+            HTTP_STORAGE_BACKEND_URL: 'https://$$cap_appname-storage.$$cap_root_domain/api/v2'
+        depends_on:
+            - $$cap_appname-room
+            - $$cap_appname-storage-backend
+    $$cap_appname-room:
+        image: excalidraw/excalidraw-room
+        restart: always
+    $$cap_appname-storage-backend:
+        image: kiliandeca/excalidraw-storage-backend:$$cap_storage_backend_version
+        restart: always
+        environment:
+            STORAGE_URI: redis://srv-captain--$$cap_appname-redis:6379
+            STORAGE_TTL: 2592000000
+        depends_on:
+            - $$cap_appname-redis
+        ports:
+            - '8080:8080'
+        caproverExtra:
+            containerHttpPort: '8080'
+    $$cap_appname-redis:
+        image: redis:$$cap_redis_version
+        restart: always
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        volumes:
+            - $$cap_appname-redis-data:/data
+
+caproverOneClickApp:
+    variables:
+        - id: $$cap_excalidraw_version
+          label: Excalidraw version
+          defaultValue: 'latest'
```

Comment:
![image](https://user-images.githubusercontent.com/10582082/192157529-41d4fc31-0d20-4bbe-8f24-90ab53313695.png)

Reply:
Actually [kiliandeca/excalidraw](https://hub.docker.com/r/kiliandeca/excalidraw/tags) is only providing latest tag **but I already mapped it with a variable** so it can be easily adapted by end user. Other suggestion?

Reply:
ah this really sucks :/ it's the dumbest way of using Docker... :man_facepalming: 

1- I suggest you open an issue with the maintainer of that Docker image and have them fix this issue.

2- As a temporary fix, you can do something like this:
https://github.com/caprover/one-click-apps/blob/48d2870e6e5769acce66a4a14712968f347683bf/public/v4/apps/mongodb-backup-s3.yml#L21

Reply:
1. I already was dreaming of doing it... It seems that it was only a dream after all :thinking: Here right now from dream to action: https://gitlab.com/kiliandeca/excalidraw-fork/-/issues/2

Reply:
2. """Nice""" workaround... even if it's really not a good solution to work like this as you said.